client/src/Components/Common/Navbar/Menu.js

```diff
@@ -126,7 +130,7 @@ export default class Menu extends PureComponent {
             {isWorker && (
               <MenuItem to={MY_REVIEWS_URL} onClick={toggleMenu}>
                 <MenuIcon icon="starComment" height="19" width="19" />
-                Reviews you've given
+                Reviews you&apos;ve given
```

Comment:
Out of curiosity, is this a better way to write an apostrophe in the code?

Reply:
I'm not sure but that was a warning from the eslint and they recommend to use this way